app/Services/Model/Imp/SummitSponsorService.php

```diff
@@ -237,6 +238,9 @@ public function addSponsorUser(Summit $summit, int $sponsor_id, int $member_id):
             if (is_null($member) || !$member instanceof Member)
                 throw new EntityNotFoundException("Member not found.");
 
+            if (!$member->belongsToGroup(IGroup::Sponsors))
```

Comment:
this validation is redundant is already done at model level
app/Models/Foundation/Summit/Sponsor.php addUser